spec/model_spec.cr

```diff
@@ -9,6 +9,29 @@ private class QueryMe < LuckyRecord::Model
   end
 end
 
+private class MissingColumn < LuckyRecord::Model
+  table users do
+    field missing : String
+  end
+end
+
+private class OptionalFieldOnRequiredColumn < LuckyRecord::Model
+  table users do
+    field name : String?
+  end
+end
+
+private class RequiredFieldOnOptionalColumn < LuckyRecord::Model
+  table users do
+    field nickname : String
+  end
+end
+
+private class MissingTable < LuckyRecord::Model
```

Comment:
Maybe `MissingButSimilarlyNamedTable`? So it's a bit more clear that it is purposely slightly misspelled?